Services/Services/ApiRequest.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Interfaces;
+using Interfaces.Models;
+
+namespace Services
+{
+    public class ApiRequest : IApiRequest
```

Comment:
Register your interface and implementation in the DI and inject it in the controller.
https://docs.microsoft.com/en-us/aspnet/core/fundamentals/dependency-injection?view=aspnetcore-5.0